pkg/controller/storagecluster/generate.go

```diff
@@ -41,3 +41,11 @@ func generateNameForCephFilesystemSC(initData *ocsv1.StorageCluster) string {
 func generateNameForCephBlockPoolSC(initData *ocsv1.StorageCluster) string {
 	return fmt.Sprintf("%s-ceph-rbd", initData.Name)
 }
+
+func generateNameForRBDSnapshotClass(initData *ocsv1.StorageCluster) string {
```

Comment:
this can be a generic name Isnt it something like `generateNameForSnapshotClass` and pass the string `rbd` or `cephfs` in the arg so that we can carve out the name in this function. @aruniiird

Reply:
Done